pkg/cli/kubernetes/portforward/application_watcher.go

```diff
@@ -60,16 +58,21 @@ func NewApplicationWatcher(options Options) *applicationWatcher {
 func (aw *applicationWatcher) Run(ctx context.Context) error {
 	defer close(aw.done)
 
-	// We use the `radapp.io/application` label to include pods that are part of an application.
-	// This can include the user's Radius containers as well as any Kubernetes resources that are labeled
-	// as part of the application (eg: something created with a recipe).
-	req, err := labels.NewRequirement(kubernetes.LabelRadiusApplication, selection.Equals, []string{aw.Options.ApplicationName})
```

Comment:
Is this refactor because the "selector" is calculated outside this code?

Reply:
exactly